x/tokenomics/keeper/settle_session_accounting.go

```diff
@@ -103,7 +103,17 @@ func (k Keeper) SettleSessionAccounting(
 	logger.Info(fmt.Sprintf("About to start settling claim for %d compute units", claimComputeUnits))
 
 	// Calculate the amount of tokens to mint & burn
-	settlementAmt = k.getCoinFromComputeUnits(ctx, root)
+	serviceID := sessionHeader.Service.Id
+	multiplier, found := k.GetChainMuliplier(ctx, serviceID)
+	if found {
+		settlementAmt = k.getCoinFromComputeUnitsByChainMultiplier(root, multiplier)
+	} else {
+		// ToDo: should this be an error rather than a warning? should chain-specific multipliers be mandatory?
```

Comment:
1. Use the appropriate TODOs: https://github.com/pokt-network/poktroll/blob/main/Makefile#L456
2. We also have this if you want a reminder to avoid merging the code in: https://github.com/pokt-network/poktroll/blob/6f2fe5b6ecb2db74572ecc32e8fbd6e3b9dc2461/.github/workflows/reviewdog.yml#L11